### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by Brunoga-MS**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. *Adds a new error message content to the existing **VM Log Alert policies fail to remediate section** in the Known Issue page*
2. *Adds a new section for **Failed to deploy because of 'location' property not specified error** in the Known Issue page
3. * Adds the **Switching between BYO UAMI and new UAMI** ** in the Bring Your Own User Assigned Managed Identity (BYO UAMI) page*

All the above are related to the USer Story [AMBA - Update Known Issues and BYO UAMI documentation](https://dev.azure.com/CSUSolEng/Azure%20Landing%20Zones/_queries/edit/35143/?triage=true)

### Breaking Changes

1. *NONE*

## As part of this Pull Request I have

- [X] Read the Contribution Guide and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/azure-monitor-baseline-alerts/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/azure-monitor-baseline-alerts/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/azure-monitor-baseline-alerts/)
- [X] Ensured PR tests are passing
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/patterns/alz/Bring-your-own-Managed-Identity.md [hunk 1]
```diff
@@ -58,16 +58,13 @@ This new feature is used in Log-search based alerts. At the moment of this relea
 We're planning to use this feature more in the future and to include it as part of other alerts.
 {{< /hint >}}
 
-<!--
 ## Switching between BYO UAMI and new UAMI
 
 The [conditional deployment behavior](../alz/Bring-your-own-Managed-Identity.md#conditional-deployment-behavior) discussed earlier, allows brownfield customers to switch from a new created UAMI to an existing one and viceversa.
 Should customers decide to switch, it will be enough to:
 
 - change the values in the parameter file to match one of the two scenarios previously discussed
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/Bring-your-own-Managed-Identity.md:66]:
```suggestion
- Change the values in the parameter file to match one of the two scenarios previously discussed
```

File: docs/content/patterns/alz/Bring-your-own-Managed-Identity.md [hunk 2]
```diff
@@ -58,16 +58,13 @@ This new feature is used in Log-search based alerts. At the moment of this relea
...
 - redeploy the AMBA-ALZ pattern
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/Bring-your-own-Managed-Identity.md:67]:
```suggestion
- Redeploy the AMBA-ALZ pattern
```

File: docs/content/patterns/alz/Bring-your-own-Managed-Identity.md [hunk 3]
```diff
@@ -58,16 +58,13 @@ This new feature is used in Log-search based alerts. At the moment of this relea
...
-- run the remediation. Atthe moment it is sufficient to run the remediation for the [Deploy Azure Monitor Vaseline Alerts for Hybrid VMs](https://raw.githubusercontent.com/Azure/azure-monitor-baseline-alerts/main/patterns/alz/policySetDefinitions/Deploy-HybridVM-Alerts.json) policy initiative
+- run the remediation for the [Deploy Azure Monitor Baseline Alerts for Hybrid VMs](https://raw.githubusercontent.com/Azure/azure-monitor-baseline-alerts/main/patterns/alz/policySetDefinitions/Deploy-HybridVM-Alerts.json) policy initiative as documented at [Remediate Policies](../deploy/Remediate-Policies)
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/Bring-your-own-Managed-Identity.md:68]:
```suggestion
- Run the remediation for the [Deploy Azure Monitor Baseline Alerts for Hybrid VMs](https://raw.githubusercontent.com/Azure/azure-monitor-baseline-alerts/main/patterns/alz/policySetDefinitions/Deploy-HybridVM-Alerts.json) policy initiative as documented at [Remediate Policies](../deploy/Remediate-Policies)
```

File: docs/content/patterns/alz/Known-Issues.md [hunk 4]
```diff
@@ -83,7 +87,7 @@ If you deployed AMBA just one time, you have 14 deployment instances
 The limit of 800 deployment for the given management group scope has been reached. More information can be found at [Management group limits](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#management-group-limits)
 
 ### Resolution
-To resolve this issue, follow the steps below:
+To resolve this issue, follow the following steps:
```
Comment (arjenhuitema) [inline docs/content/patterns/alz/Known-Issues.md:90]: follow the following. Maybe this works better?

```suggestion
To resolve this issue, follow these steps:
```

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Thanks @Brunoga-MS, few comments, can you review? After that we should be good the merge!

File: docs/content/patterns/alz/Known-Issues.md (hunk 4 above)
Comment (Brunoga-MS) [reply docs/content/patterns/alz/Known-Issues.md:90]: This one is coming from Acrolinx spell and grammar check. We should not change it

**Review by arjenhuitema — APPROVED** [review]
LGTM.

**Pull request merged**